topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -148,19 +146,17 @@ The size of scRNA files (.fastq) are typically in the gigabyte range and are som
 {: .hands_on}
 
 
-For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*](../scrna-plates-batches-barcodes/slides.html) slides.
+For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*]({{ site.baseurl }} topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html %}) slides.
```

Comment:
is there a / missing and a whitespace to much?

Reply:
not seeing it, where is the whitespace?

Reply:
That was a comment for @bebatut. Shoudn't it be `{{ site.baseurl }}/topics/transcriptomics/tutorials/s...`

Reply:
```suggestion
For a more detailed understanding of the naming conventions used in generating our FASTQ pair, please consult the [*Plates, Batches, and Barcodes*]({{ site.baseurl }}{% link topics/transcriptomics/tutorials/scrna-plates-batches-barcodes/slides.html %}) slides.
```

Reply:
I aldready fixed it in my last commit. Thanks @bgruening and @nsoranzo for catching it